packages/person/src/PersonSelect.tsx

```diff
@@ -48,19 +47,7 @@ export const PersonSelect: React.FC<PersonSelectProps> = ({
   children,
   ...props
 }: PropsWithChildren<PersonSelectProps>) => {
-  const searchRef = useRef<HTMLPersonSelectCustomElement>(null);
-
-  useEffect(() => {
-    for (const [name, value] of Object.entries(extractProps<ElementProps>(props))) {
-      if (searchRef.current) {
-        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-        // @ts-ignore
-        searchRef.current[name] = value;
-      }
-    }
-  }, [props]);
-
-  return <PersonSelectComponent ref={searchRef}>{children}</PersonSelectComponent>;
```

Comment:
I thought we needed this when I worked with the person select component to get it working for both the code and storybook? Is this wrong, @eikeland and @odinr ?

Reply:
It was not needed and the extended events did not work when you add them by ref.
it is working now.